docroot/themes/humsci/su_humsci_theme/Gruntfile.js

```diff
@@ -113,7 +113,7 @@ module.exports = function(grunt) {
           // PATTERNS
           'patterns/molecules/date-stacked-vertical-card/css/date-stacked-vertical-card.css': 'patterns/molecules/date-stacked-vertical-card/scss/date-stacked-vertical-card.scss',
           'patterns/molecules/horizontal-card/css/horizontal-card.css':                       'patterns/molecules/horizontal-card/scss/horizontal-card.scss',
-          'patterns/organisms/masonry/css/masonry.css':                                       'patterns/organisms/masonry/scss/masonry.scss',
```

Comment:
We probably need the organism for masonry too. there are scss in the masonry pattern.

Reply:
That seems like it would make sense. :)